TYPO3.Flow/Classes/TYPO3/Flow/Security/Cryptography/RsaWalletServicePhp.php

```diff
@@ -65,6 +72,10 @@ public function injectSettings(array $settings)
         } else {
             throw new MissingConfigurationException('The configuration setting TYPO3.Flow.security.cryptography.RSAWalletServicePHP.keystorePath is missing. Please specify it in your Settings.yaml file. Beware: This file must not be accessible by the public!', 1305711354);
         }
+
+        if (isset($settings['security']['cryptography']['RSAWalletServicePHP']['paddingAlgorithm'])) {
```

Comment:
This will always yield true even if the `OPENSSL_PKCS1_OAEP_PADDING` constant doesn't exist

Reply:
Dang, right, Flow will leave the constant unreplaced and the setting will come through as `%WHATEVER_IT_IS%`

Reply:
@kdambekalns This should not have been marked resolved then, right?

Reply:
Ah, yes it should.. I got confused